data/uking_functions.csv

```diff
@@ -86988,17 +86988,17 @@
 0x0000007101051410,sub_7101051410,228,
 0x00000071010514f4,sub_71010514F4,16,
 0x0000007101051504,sub_7101051504,56,
-0x000000710105153c,_ZN3aal24FxReverbI3DL2ParamSetterC2Ev,52,
-0x0000007101051570,sub_7101051570,84,
-0x00000071010515c4,_ZN3aal24FxReverbI3DL2ParamSetter7setRoomEif,16,
-0x00000071010515d4,_ZN3aal24FxReverbI3DL2ParamSetter9setRoomHfEi,12,
-0x00000071010515e0,_ZN3aal18FxDelayParamSetter8setDelayEf,12,
-0x00000071010515ec,_ZN3aal24FxReverbI3DL2ParamSetter14setReflectionsEi,12,
-0x00000071010515f8,_ZN3aal18FxDelayParamSetter11setFeedbackEf,12,
-0x0000007101051604,sub_7101051604,1272,
-0x0000007101051afc,nullsub_4419,4,
-0x0000007101051b00,nullsub_4420,4,
-0x0000007101051b04,sub_7101051B04,96,
+0x000000710105153c,_ZN3aal24FxReverbI3DL2ParamSetterC2Ev,52,_ZN3aal24FxReverbI3DL2ParamSetterC2Ev
```

Comment:
This generates a lot of check.py warnings:

https://ci.botw.link/#/builders/2/builds/66/steps/6/logs/warnings__58_

It also causes these functions to be treated as decompiled (as far as progress.py is concerned). I think we should update the default names (the second column) rather than the last column, which is only used for functions that have been decompiled.